peer/example_test.go

```diff
@@ -38,23 +39,29 @@ func mockRemotePeer() error {
 			return
 		}
 
-		// Create and start the inbound peer.
-		p := peer.NewInboundPeer(peerCfg)
-		p.AssociateConnection(conn)
+		// Create and start the inbound
+		p := NewInboundPeer(peerCfg)
+		err = p.AssociateConnection(conn)
+		if err != nil {
+			fmt.Printf("AssociateConnection: error %+v\n", err)
+			return
+		}
 	}()
 
 	return nil
 }
 
 // This example demonstrates the basic process for initializing and creating an
-// outbound peer. Peers negotiate by exchanging version and verack messages.
+// outbound  Peers negotiate by exchanging version and verack messages.
 // For demonstration, a simple handler for version message is attached to the
-// peer.
+//
```

Comment:
The above changes seem to be a `replace` error.

Reply:
LOL, yeah, I moved the whole thing out of peer_test, so had to remove all `peer.`